01_fibonacci/fibi.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+//prints the fibonacci series
+func fibonacci(n int) {
+	a, b := 0, 1
+
+	for i := 0; i <= n; i++ {
+		if a >= 1 {
+			fmt.Fprintln(out, a)
```

Comment:
better to decouple fibonacci sequence calculation from IO

https://github.com/anz-bank/go-training/issues/37

Reply:
@anz-rfc it's good suggestion.It can be done using arrays to decouple fibonacci sequence calculation from IO, after completing the arrays lesson, will re work on this